cli/js/tests/event_target_test.ts

```diff
@@ -151,18 +151,18 @@ unitTest(function eventTargetThisShouldDefaultToWindow(): void {
   };
 
   addEventListener("hello", listener);
-  window.dispatchEvent(event);
+  self.dispatchEvent(event);
   assertEquals(n, 2);
   n = 1;
   removeEventListener("hello", listener);
-  window.dispatchEvent(event);
+  self.dispatchEvent(event);
   assertEquals(n, 1);
 
-  window.addEventListener("hello", listener);
+  self.addEventListener("hello", listener);
   dispatchEvent(event);
   assertEquals(n, 2);
   n = 1;
-  window.removeEventListener("hello", listener);
+  self.removeEventListener("hello", listener);
   dispatchEvent(event);
   assertEquals(n, 1);
 });
```

Comment:
Merge with master, hopefully this doesn't conflict with his recent Event changes (fc4819e1e0624b9377913932202bc31f5a25bab4)